AUTHORS

```diff
@@ -186,7 +186,6 @@ Shido <the.keikun@gmail.com>
 Shizuke <matiasdvt@hotmail.com>
 sigtus <vantimothy@openmailbox.org>
 sirius <fw@f-ws.de>
-Sirius <zopokx@gmail.com>
```

Comment:
The _AUTHORS_ files is generated from commit logs. If you want to "rename" yourself please use the [.mailmap](https://github.com/HerculesWS/Hercules/blob/stable/.mailmap) file.